packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.js

```diff
@@ -0,0 +1,83 @@
+import { LightningElement, api, track } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+import { getItemFromLocalStorage, setItemInLocalStorage } from 'c/quanticUtils';
+import recentQueriesLabel from '@salesforce/label/c.quantic_RecentQueries';
+import emptyListLabel from '@salesforce/label/c.quantic_EmptyRecentQueriesLabel';
+
+export default class QuanticRecentQueries extends LightningElement {
+  labels = {
+    recentQueriesLabel,
+    emptyListLabel,
+  }
+  
+  /** @type {import("coveo").RecentQueriesState} */
+  @track state;
+  /** @type {string} */
+  @api label = this.labels.recentQueriesLabel;
+  /** @type {string} */
+  @api engineId;
+  /** @type {import("coveo").RecentQueriesList} */
+  recentQueriesList;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isCollapsed = false;
+  /** @type {string} */
+  collapseIcon = 'utility:dash';
+  /** @type {number} */
+  @api maxLength = 10;
```

Comment:
for readability i think it would be better to continue to group the @api declarations together.